tools/clang/test/CodeGenSPIRV/intrinsics.length.hlsl

```diff
@@ -0,0 +1,28 @@
+// Run: %dxc -T vs_6_0 -E main
+
+// According to HLSL reference:
+// The 'length' function can only operate on vector of floats.
+
+// CHECK:      [[glsl:%\d+]] = OpExtInstImport "GLSL.std.450"
+
+void main() {
+  float result;
+
+// CHECK:      [[a:%\d+]] = OpLoad %float %a
+// CHECK-NEXT: [[length_a:%\d+]] = OpExtInst %float [[glsl]] Length [[a]]
+// CHECK-NEXT: OpStore %result [[length_a]]
+  float a;
+  result = length(a);
```

Comment:
Interesting case. Semantic analysis doesn't error out. But this is a scalar... The doc says only vector...